OracleFMWInfrastructure/dockerfiles/README.md

```diff
@@ -0,0 +1,135 @@
+Oracle Fusion Middleware Infrastructure on Docker
```

Comment:
I don't think you need to duplicate the `README.md` in two locations, because they could get out of sync.